### ISSUE TITLE
windows console.log not showing until program ended

### ISSUE THREAD

**Issue by camhart** [electron/fiddle#699]
On windows 10, using fiddler 0.23.1, console.log's in the main process don't show in the console output until you end the program.

**Comment by erickzhao** [comment]
Might be related to https://github.com/electron/fiddle/pull/676

**Comment by codebytere** [comment]
@camhart could you please provide a reproducible fiddle to run that demonstrates the problem? I can't reproduce this and will not be able to move forward without that.

**Comment by camhart** [comment]
Sure...
```
// Modules to control application life and create native browser window
const {app, BrowserWindow} = require('electron')
const path = require('path')

console.log(`hi there at ${new Date().toLocaleString()}`) //doesn't show until program is ended...

function createWindow () {
  // Create the browser window.
  const mainWindow = new BrowserWindow({
    width: 800,
    height: 600,
    webPreferences: {
      preload: path.join(__dirname, 'preload.js')
    }
  })

  // and load the index.html of the app.
  mainWindow.loadFile('index.html')

  // Open the DevTools.
  // mainWindow.webContents.openDevTools()
}

// This method will be called when Electron has finished
// initialization and is ready to create browser windows.
// Some APIs can only be used after this event occurs.
app.whenReady().then(() => {
  createWindow()
  
  app.on('activate', function () {
    // On macOS it's common to re-create a window in the app when the
    // dock icon is clicked and there are no other windows open.
    if (BrowserWindow.getAllWindows().length === 0) createWindow()
  })
})

// Quit when all windows are closed, except on macOS. There, it's common
// for applications and their menu bar to stay active until the user quits
// explicitly with Cmd + Q.
app.on('window-all-closed', function () {
  if (process.platform !== 'darwin') app.quit()
})

// In this file you can include the rest of your app's specific main process
// code. You can also put them in separate files and require them here.

```

See timestamps in screenshot.

![image](https://user-images.githubusercontent.com/3038809/120984933-fa337a00-c72f-11eb-9f16-3c33b1de8bb5.png)

**Comment by Juniourrau** [comment]
I am also facing the same issue as @camhart 
https://gist.github.com/a4f032ac0bfcc8392f4dca2e0a476cbe

Here is a gist that @codebytere can use to see the bug

**Comment by nornagon** [comment]
@Juniourrau Can you comment here with the version of Electron Fiddle you're using? #676 landed in v0.23.0, so it would be useful to know whether or not that might have fixed the issue.

**Comment by Juniourrau** [comment]
@nornagon I am using v0.23.2.0 which I assume is the latest one.
